### PR TITLE
New(job): OpenDataHub - Model Registry standalone

### PR DESCRIPTION
**Pull request by rkubis**
New job for testing standalone deployment of OpenDataHub - Model Registry component.

### CODE REVIEW

File: ci-operator/config/opendatahub-io/model-registry/opendatahub-io-model-registry-main.yaml [hunk 1]
```diff
@@ -38,6 +38,48 @@ tests:
...
+          # Install Python client for model-registry
+          pip3.9 install -e clients/python
+          # Change URL for tests
+          sed -i -e "s/localhost/model-registry-service/" test/robot/*.resource test/robot/*.py
+          # Change image of model-registry deployment
+          sed -i -e "s|image: quay.io/opendatahub/model-registry.*|image: ${ODH_MODEL_REGISTRY_IMAGE}|" manifests/kustomize/base/model-registry-deployment.yaml
+          # Create namespace for tests
+          oc create namespace kubeflow
+          # Deploy model-registry instance
+          oc apply -k manifests/kustomize/overlays/postgres
```
Comment (tarilabs) [inline ci-operator/config/opendatahub-io/model-registry/opendatahub-io-model-registry-main.yaml:67]: I'd rather opt for the mysql/mariadb one, given recent discussions

Comment (rkubis) [reply ci-operator/config/opendatahub-io/model-registry/opendatahub-io-model-registry-main.yaml:67]: I would like to test all variants later too 👍

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-model-registry-standalone pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo pull-ci-opendatahub-io-model-registry-main-images

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-model-registry-standalone pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo pull-ci-opendatahub-io-model-registry-main-images

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-model-registry-standalone pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo pull-ci-opendatahub-io-model-registry-main-images

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-model-registry-standalone pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo pull-ci-opendatahub-io-model-registry-main-images

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-model-registry-standalone pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo pull-ci-opendatahub-io-model-registry-main-images

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-model-registry-standalone pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo pull-ci-opendatahub-io-model-registry-main-images

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-model-registry-standalone pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo pull-ci-opendatahub-io-model-registry-main-images

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-model-registry-standalone pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo pull-ci-opendatahub-io-model-registry-main-images

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-model-registry-standalone pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo pull-ci-opendatahub-io-model-registry-main-images

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-model-registry-standalone pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo pull-ci-opendatahub-io-model-registry-main-images

**Pull request closed without merging**